C01/project-metadata/C01-ProjectId612-metadata.json

```diff
@@ -69,12 +69,44 @@
       }
     },
     "type": "Feature",
+    "place_name": "Taita–Taveta, Kenya",
     "geometry": {
       "type": "Point",
       "coordinates": [
         38.802647,
         -3.910022
       ]
     }
+  },
+  "regen:previewPhoto": {
+    "@type": "schema:URL",
+    "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/kasigau.png"
+  },
+  "regen:galleryPhotos": {
+    "@list": [
+        {
+            "@type": "schema:URL",
+            "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/image1.png"
+        },
+        {
+            "@type": "schema:URL",
+            "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/image2.png"
+        },
+        {
+            "@type": "schema:URL",
+            "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/image3.png"
+        },
+        {
+            "@type": "schema:URL",
+            "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/image4.png"
+        }
+    ]
+  },
+  "regen:landStory": "For years, the land between Tsavo East and Tsavo West National parks in Kenya served both as home to a slowly failing cattle ranch and as the main migration corridor for local wildlife moving between the two National Parks. When we first encountered Rukinga, the community and the wildlife were at odds. Rukinga was a bruised, balding land, barren of wildlife. Cattle had grazed the fields into dust, poachers slipped on and off the ranch with ease, and trees were being clear cut along the area’s critical rainwater basin.",
```

Comment:
I'll delete this, in favor of `regen:description`.

Do we need 'landStewardStory' and `landStewardStoryTitle` anymore? I may have been mistaken to add these back here. @S4mmyb 

```suggestion
```

Reply:
I think we can keep them! I just changed `landStory` to `description` because I think it would be a bit more all encompassing for projects in the long run (thinking about credit classes/projects without much story). But I think `landStewardStory` and `landStewardStoryTitle` make the most sense here